src/net.h

```diff
@@ -689,6 +689,8 @@ class CNode
     bool m_limited_node{false}; //after BIP159, set by version message
     const bool fInbound;
     std::atomic_bool fSuccessfullyConnected{false};
+    // Setting fDisconnect to true will cause the node to be disconnected the
+    // next time DisconnectNodes() runs
```

Comment:
nit: would be nice to use doxygen compatible comments (see developer notes and https://dev.visucore.com/bitcoin/doxygen/class_c_node.html#adebd9bb3224ed2c5ab63fdfd9c6e9074)